addons/mass_mailing/static/src/components/message/message.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+    <t t-inherit="mail.Message" t-inherit-mode="extension">
+        <xpath expr="//t[@t-if][./div/t[@t-if][./span/i[hasclass('o_Message_notificationIcon')][@name='failureIcon']]]" position="attributes">
```

Comment:
all these selectors are way too complex, make sure to target a single class/name that is easy to grep